diff/moosedeepdiff.py

```diff
@@ -3,19 +3,12 @@
 from deepdiff import DeepDiff
 
 class MOOSEDeepDiff(DeepDiff):
-    def __init__(self, t1, t2, relative_error=None, absolute_error=None, **kwargs):
-        kwargs_list = [value for key, value in kwargs.items()]
-        kwargs_list = sorted(kwargs_list)
-        super().__init__(t1, t2, *kwargs_list)
+    def __init__(self, *args, relative_error=None, absolute_error=None, **kwargs):
         self.rel_err = relative_error
         self.abs_err = absolute_error
```

Comment:
Your original format for this was correct after all, I think the problem you were having is that in your compare function you forgot to use this class.